Tests/SwiftSyntaxBuilderTest/Assertions.swift

```diff
@@ -19,10 +19,13 @@ func assertBuildResult<T: SyntaxProtocol>(
   _ buildable: T,
   _ expectedResult: String,
   trimTrailingWhitespace: Bool = true,
+  format: Bool = true,
   file: StaticString = #file,
   line: UInt = #line
 ) {
-  var buildableDescription = buildable.formatted().description
+  var buildableDescription = format
+    ? buildable.formatted().description
+    : buildable.description
```

Comment:
@ahoppen do you remember why you changed this to format by default? Makes all these basic format tests too, which maybe we want, but it does end up meaning we miss things like this.

Reply:
I don’t remember. But I believe it was something along the lines of: You are using `SwiftSyntaxBuilder` to build your syntax tree and if you are doing that, it’s expected that you call `.formatted` in the end go get something that actually parses again, so that’s what we’re testing here.